src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -127,6 +322,33 @@ public void putCreateExternalBinaryContentProxy(final String uri) {
                         "request entity, as indicated by a link with " +
                         "rel=\"http://fedora.info/definitions/fcrepo#ExternalContent\"",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final String location = uri + info.getId();
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUri, "proxy")));
+        doPutUnverified(location, headers).then()
+                .statusCode(201);
+    }
+
+    /**
+     * 3.9-B-3b PutCreate Proxy File URI
+     *
+     * @param uri of base container of Fedora server
+     */
+    @Test(groups = { "SHOULD" })
+    @Parameters({ "param1" })
+    public void putCreateExternalFileUriBinaryProxy(final String uri) {
+        final TestInfo info = setupTest("3.9-B-3b", "putCreateExternalFileUriBinaryProxy",
+                "Fedora servers should support the creation of LDP-NRs with content external to the " +
+                        "request entity, as indicated by a link with " +
+                        "rel=\"http://fedora.info/definitions/fcrepo#ExternalContent\"",
+                "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final String location = uri + info.getId();
```

Comment:
It would be better if you check to see if the base URL does not end with a `/`. If so add in a slash.